packages/core/src/resolver.js

```diff
@@ -10,6 +11,24 @@ import {
   clearRequireCache
 } from '@nuxt/utils'
 
+// Polyfill Node's `Module.createRequire` / `Module.createRequireFromPath` function if not present
+export const createRequire =
```

Comment:
Nice! Just can we maybe add to utils? (or ideally an npm package for polyfill import :p)

Reply:
There's currently no standalone npm package for `createRequire`, but there is [a Polyfill](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-core-utils/src/create-require-from-path.ts) in `gatsby-core-utils`, which, funny enough, was added by @arcanis (https://github.com/cstoddart/gatsby/commit/72e0d6fb874cfd2d355265f08ca08460dcde873c#diff-4e10856144ca00b229a8029e440b9e60) so it is the exact same one I used here :D.
We have 3 options:
- duplicate the Polyfill in `@nuxt/utils`
- depend on `gatsby-core-utils`
- maybe @arcanis would like to publish the polyfill as a standalone npm package (What do you think?)

Reply:
Not sure it's worth adding in the Yarn monorepo since it's a quite "static" tool, which won't need updates going forward 🤔

Reply:
Adding it as an external package in few minutes :)

Reply:
https://www.npmjs.com/package/create-require